github/files/drupal7/scripts/devops/reinstall/post_settings.yml

```diff
@@ -23,6 +23,18 @@
   when: item.status == true
   with_items: env.post_settings
 
+- name: Checking a domain_access is enabled
+  shell: 'drush pml --status=enabled -l {{ site_url }} | grep -c "(domain)" || true'
```

Comment:
Regadring enabling domain module
Here is a list of queue  https://github.com/propeoplemd/cibox/blob/master/github/files/drupal7/scripts/devops/reinstall/vars/environments/global_env.yml commands for cibox
We have 
1. pre_setting
2. modules enabling
3. drush commands
4. post settings
This is a turn how they are executed one by one. If we need something before pre_settings - let's create a use case and think about this with @Sanchiz